backend/app.py

```
@@ -0,0 +1,23 @@
+from flask import Flask, send_from_directory
```

Ruff (0.12.2)

`flask.send_from_directory` imported but unused

Remove unused import: `flask.send_from_directory`

(F401)